k6.js

```diff
@@ -0,0 +1,26 @@
+import http from 'k6/http';
+import { check } from 'k6';
+
+export let options = {
+  stages: [
+    { duration: '30s', target: 10 },
+    { duration: '30s', target: 20 },
+    { duration: '30s', target: 40 },
+    { duration: '30s', target: 80 },
+    { duration: '30s', target: 160 },
+    { duration: '30s', target: 0 }
+  ]
```

Comment:
Interesting use of stages. I didn't know you could change parameters at each stage.